packages/block-library/src/navigation-menu/edit.js

```diff
@@ -173,6 +240,7 @@ export default compose( [
 			orderby: 'id',
 		};
 		return {
+			hasExistingNavItems: !! hasExistingNavItems,
```

Comment:
This can introduce confusion, since the "exported" prop is a Boolean while the inner one is an integer. Instead, the inner variable should be a Boolean from the start, i.e. in its assignment it should read `.length > 0` or equivalent.

Reply:
I've simplified this now.